docs/Cerberus.md

```diff
@@ -0,0 +1,39 @@
+# Cerberus
+
+[Cerberus](https://www.cl.cam.ac.uk/~pes20/cerberus/) offers executable semantics for a substantial fragment of C and
+CHERI-C languages. It is implemented via an elaboration into a simpler Core language, which is displayed as the compiler
+output in the Compiler Explorer. Evaluation of C programs (execution) is also supported.
+
+## Prerequisites
+
+The easiest way to install both the Cerberus and Cerberus-CHERI compilers is by using Docker:
```

Comment:
This is an interesting document but I don't see how it relates to Compiler Explorer. CE can't launch compilers in a docker environment, so this section seems unhelpful?